.github/scripts/test-project.sh

```diff
@@ -67,6 +67,18 @@ fi
 echo "$ cd $dir/$project"
 cd "$dir/$project"
 
+# if FORCE_CUSTOM_OUTPUT is set, we execute commands that will turn the project into a custom output project
```

Comment:
```suggestion
# if FORCE_PRISMA_CLIENT_CUSTOM_OUTPUT is set, we execute commands that will turn the project into a custom output project
```
Could you rename it to something like this, maybe?

`FORCE_CUSTOM_OUTPUT` is a bit too generic I think, it could be about many things.